CAIPs/caip-122.md

```diff
@@ -37,7 +37,7 @@ The data model _MUST_ contain the following fields:
 | Name              | Type            | Mandatory | Description                                                                                                                                                                                       |
 | ----------------- | --------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `domain`          | string          | ✓         | [RFC 4501][rfc 4501] `dnsauthority` that is requesting the signing.                                                                                                                               |
-| `address`         | string          | ✓         | Blockchain address, as defined by [CAIP-10][caip-10], performing the signing; should include [CAIP-2][caip-2] chain id namespace                                                                  |
+| `address`         | string          | ✓         | Blockchain address performing the signing, expressed as the `account_id` segment of a [CAIP-10][caip-10]; should NOT include [CAIP-2][caip-2] `chain_id`.| namespace                                                                  |
```

Comment:
This change is not consistent with other changes made in https://github.com/ChainAgnostic/CAIPs/pull/236/files (where string representation example depends on `address` being a CAIP-10 identifier),

Rather than splitting CAIP-10 identifier into address and chain ID, I suggest renaming `address` field to `account_id`.

Reply:
this breaks our current implementation, but i guess is fine as CAIP-10 itself was updated and doesn't call it `address` anymore.